README.md

```diff
@@ -1,2 +1,180 @@
-# AutoAI-code-generation
-code pattern to show notebook generation feature of AutoAI
+
+<!-- Add a new Title and fill in the blanks -->
+# Generate Python notebook for pipeline models using AutoAI
+
+## Summary
+
+In this code pattern, we will learn how to automatically generate a jupyter notebook model through AutoAI. We will explore, modify and retrain this model pipeline using python code. Lastly, we will deploy this model in Watson Machine Learning using WML APIs.
+
+## Description
+
+AutoAI is a graphical tool available within Watson Studio that analyzes your dataset, generates several model pipelines  and ranks them based on the metric chosen for the problem. This code pattern shows extended features of AutoAI. More basic AutoAI exploration for the same dataset is covered in the [Generate machine learning model pipelines to choose the best model for your problem](https://developer.ibm.com/tutorials/generate-machine-learning-model-pipelines-to-choose-the-best-model-for-your-problem-autoai/) article.
+
+When you have completed this code pattern, you will understand how to:
+
+* Run an AutoAI experiment.
+* Generate and save a Python notebook.
+* Execute notebook and analyse results.
+* Improvise pipeline model and retrain using auto-generated notebook.
```

Comment:
not sure what you mean here w/ Improvise.  Did you mean "Improve".  Improvise just doesn't seem to fit for me.

Reply:
changed it to - * Make changes and retrain model using Watson Machine Learning SDKs.